nest/03.advanced_concepts.md

```diff
@@ -0,0 +1,70 @@
+# 3 Advanced concepts
+In this chapter, we will detail some advanced concepts related to `nest`, which will help understanding how `nest` works, but aren't strictly needed for a daily use of `nest`.
+On the other hand, readers interested in `finest` should read this chapter, because those concepts are important to understand how `finest` works.
+
+## 3.1 Managing repositories
+### 3.1.1 The official repositories
+The official repositories are the ones maintained by the Raven-OS team.
+For now, only one is available: `stable`, which is the one enabled by default when installing Nest.
+It contains all the stable and safe-to-use packages for Nest, and is maintained by the Raven-OS team.
+
+### 3.1.2 The unofficial repositories
+There are also unofficial repositories for Nest.
+They are not being maintained by the Raven-OS team or someone trusted by the team.
+You must decide whether to trust their maintainers and take full responsibility for any consequences of using any unofficial repository.
+
+### 3.1.3 Configuring repositories
+The repositories configuration is in the `config.toml` file of Nest.
+It would look something like this:
+
+```toml
+# Stable repository
+[repositories.stable]
+mirrors = ["https://stable.raven-os.org"]
+
+# Beta repository (uncomment to enable)
+# [repositories.beta]
+# mirrors = ["https://beta.raven-os.org"]
+
+# Nightly repository (uncomment to enable)
+# [repositories.nightly]
+# mirrors = ["https://nightly.raven-os.org"]
+```
+
+A mirror is basically a copy of all datas from the main server.
+They exist for redundancy and speed.
+Mirrors are spread all over the globe in strategic locations.
+If you live in Sweden and use a mirror from Sweden, the download speed should be faster than if you used a mirror from Australia for example.
+The point of mirrors is to choose the nearest one from your location, so you'll download packages as fast as possible.
+It's advised to have several mirrors for each repository Nest uses, because a mirror can be down or broken for a small period of time, or just be down forever.
+If Nest can't reach a mirror, it will try with the next one in the list.
+
+Only `stable` is enabled by default, so you'll have to uncomment the other repositories and their mirrors in order to use them.
+
+You can also add an unofficial repository to this list, by following the same syntax shown above in the code excerpt.
+
+## 3.2 Overview of the cache
+Nest is using a cache to store a copy of the list of all available packages for Nest on the OS.
+Nest can only install packages present in your cache.
+That's why you need to **keep it up-to-date as often as possible**, otherwise Nest won't install new versions of packages that came out on Nest's server.
```

Comment:
Shouldn't it rather be "make sure it is up to date when doing an operation" or something like that?
Keeping it up to date when doing no operation seems useless.